charts/umbrella/amazon/values.yaml

```diff
@@ -0,0 +1,57 @@
+# The amazon chart is a parent chart that installs
+# all of the other charts of Rancher PAYG offering,
+# but doesn't actually deploy any services itself.
+
+#        
+# Rancher Prime service
+#
+#
+
+# Override any settings for the Rancher prime service here
+rancher:
+        
+  installCRDs: true
+  # The Fully Qualified Domain Name for your Rancher Server
+  hostname: "rancher.my.org"
+  global:
+    cattle:
+      psp:
+        enabled: false
+  # Set a bootstrap password. If leave empty, a random password will be generated.      
+  bootstrapPassword: "rancher"
```

Comment:
I think the bootstrapPassword is randomly generated if absent. Setting a default one will be problematic from security standpoint.